schema.sql

```diff
@@ -0,0 +1,12 @@
+-- Create the table in the specified schema
+
+CREATE TABLE dbo.animal
+(
+    id INT NOT NULL PRIMARY KEY, -- primary key column
+    name [NVARCHAR](50) NOT NULL,
+    date_of_birth [NVARCHAR](50) NOT NULL,
+    escape_attempts [NVARCHAR](50) NOT NULL,
+    neutered [NVARCHAR](50) NOT NULL,
+    weight_kg [NVARCHAR](50) NOT NULL
+    -- specify more columns here
```

Comment:
- [ ] Please kindly update your data types to match those specified in the project requirements. See screenshot below for clarifications. 

![dataTypes](https://user-images.githubusercontent.com/62849872/155470942-91d1e131-d427-4841-bdc5-5c792fe99135.png)